db/migrations/20221206074145_create_users_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+ CREATE TABLE users (
+ id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL UNIQUE,
+    password VARCHAR(100) NOT NULL,
+    phone_number VARCHAR(20) NOT NULL UNIQUE,
+    point BIGINT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+    PRIMARY KEY(id) 
+ );
```

Comment:
- id 컬럼 부분 들여쓰기를 수정하여 가독성을 높여주세요!
- id 컬럼에 PRIMARY KEY 속성을 지정하는 방법 중 하나로 아래 예시 코드를 확인해주세요~!
- 암호화를 한 password를 저장하는 경우 데이터의 타입을 binary로 지정할 수 있습니다. bcrypt 모듈을 통해 암호화 된 패스워드의 최대 길이가 몇인지 다양한 소스를 통해 확인해보면 좋을 것 같습니다~!
- point의 경우 BIGINT가 아닌 decimal로 지정해야할 것 같습니다. decimal 필드에 대해 MySQL 공식문서에서 확인해보고 적용해주세요! 또한 기본값으로 point를 주어야 추후 결제API를 구현할 때 해당 포인트에서 결제 금액만큰 차감할 수 있을 것 같습니다.
- unique 속성은 constraint 제약 조건을 통해 지정하는 방식도 고려해주세요~! 제약 조건의 이름을 지정하여 추후에 관리하기 용이하기 때문입니다.
- 
```suggestion
 CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    name VARCHAR(100) NOT NULL,
    email VARCHAR(100) NOT NULL,
    password binary(60) NOT NULL,
    phone_number VARCHAR(20) NOT NULL,
    point DECIMAL(8,2) NULL DEFAULT 10000000,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
    CONSTRAINT users_email_ukey UNIQUE(email),
    CONSTRAINT users_phone_number_ukey UNIQUE(phone_number),
 );
```